metrics_utility/test/ccspv_reports/test_CCSP.py

```diff
@@ -224,16 +223,29 @@
 def test_command(cleanup):
     """Build xlsx report using build command and test its contents."""
 
-    python_executable = sys.executable
-    result = subprocess.run(
-        [python_executable, 'manage.py', 'build_report', '--month=2024-02', '--force'],
-        text=True,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        env=env_vars,
-    )
+    run_build_ext(env_vars, ['--month=2024-02', '--force'])
```

Comment:
what is `ext` meant to be in this case?

Reply:
external .. (vs int for internal) .. but open to better names :) (cli vs command didn't sound clear enough :))